plasma_framework/migrations/20_deploy_plasma_framework.js

```diff
@@ -10,11 +10,15 @@ module.exports = async (
     // eslint-disable-next-line no-unused-vars
     [deployerAddress, maintainerAddress, authorityAddress],
 ) => {
+    const isExperiment = process.env.EXPERIMENT || false;
+    const initialImmuneExitGames = (isExperiment)
+        ? config.experimental.frameworks.initialImmuneExitGames : config.frameworks.initialImmuneExitGames;
```

Comment:
What about just swapping the used config?
This works only for this additional exit game, but if we switched the whole config, then whatever changes in the future - no changes are required.